### DISCUSSION TITLE
New Rule - Do you use GitHub Discussions to suggest new Ideas?

### DISCUSSION THREAD

**Discussion by SamBWagner** [SSWConsulting/SSW.Rules.Content#4810] [Chewing the Fat]
In our Rules to Better GitHub, there's no mention of Discussions! 😱

I think we should have a rule about using them to suggest new Rules or larger Rules overhauls/changes.

Or otherwise chat about the project 🗣️

**Comment by SamBWagner** [comment]
Some comments!

**Comment by bradystroud** [comment]
Good idea, I think there is often confusion around when to use issues or discussions. 

This rule might need an update as well https://www.ssw.com.au/rules/discuss-the-backlog/#for-other-topics-brainstorm-ideas-general-discussion-etc

**Comment by danielmackay** [comment]
Love this idea @SamBWagner !

**Comment by GordonBeeming** [comment]
@SamBWagner  I think we should create a feedback type repo similar to GitHub and have all internal feedback there across all projects, GitHub tried micro feedback channels, it doesn't work. Maybe this could be split into internal and external, right now though it might be worth having that internal only to cut our teeth
https://github.com/community/community/discussions 

edit: they used to run this as https://github.com/github/feedback but moved to community/community

**Reply by GordonBeeming** [reply]
Some benefits of this are

- Less time consuming to setup new projects (it's just another topic, changes to how we generally 'run discussions is done in 1 place)
- Easier visibility on new topics (not scrolling through repos)
- All feedback is in 1 place encouraging collaboration. (people generally aren't going to just navigate to different repo discussions to see if there is something new but a little scroll through some discussions feels reasonable
- Builds community (even if just internally)
- If discussions turn to issues you are able to move them to another repo so that functionality isn't lost

**Comment by GordonBeeming** [comment]
Testing something new
